manifest.json

```diff
@@ -1,24 +1,44 @@
 {
   "manifest_version": 3,
   "name": "Hide Amazon Cart",
-  "version": "0.0.4",
+  "version": "0.0.6",
```

Comment:
This is 6 instead of 5 because I signed the extension for Firefox and then made a small change. I couldn't update the extension for Firefox without incrementing the version (or at least I didn't realize it was possible if I could).